### PR TITLE
LPD-32124 - Make sure format respects ignored files

### PR DESCRIPTION
**Pull request by bryceosterhaus**

### CODE REVIEW

File: modules/_node-scripts/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
 		"jest-environment-jsdom-thirteen": "1.0.1",
 		"jest-fetch-mock": "^3.0.3",
 		"liferay-theme-tasks": "11.5.1",
+		"micromatch": "4.0.2",
```
Comment (bryceosterhaus) [inline modules/_node-scripts/package.json:37]: This is the same matching package that fast-glob uses, so we aren't actually expanding our dependency footprint here

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
sent in this PR https://github.com/brianchandotcom/liferay-portal/pull/152645

**Pull request closed without merging**